requirements.txt

```diff
@@ -1,8 +1,8 @@
 descartes
-geopandas
+geopandas>=1.0.0
 matplotlib
 scikit-learn
-numpy
+numpy<2
```

Comment:
Is this needed?

Reply:
Yes, for now - `pylibtiff` throws me errors when `numpy` has the newest version.

Reply:
Okay!